homeassistant/components/sensor/dwd_warnapp.py

```diff
@@ -0,0 +1,245 @@
+"""
+Support for getting statistical data from a DWD-Warnapp system.
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.dwd_warnapp/
+
+Data is fetched from DWD:
+https://rcccm.dwd.de/DE/wetter/warnungen_aktuell/objekt_einbindung/objekteinbindung.html
+
+Warnungen vor extremem Unwetter (Stufe 4)
+Unwetterwarnungen (Stufe 3)
+Warnungen vor markantem Wetter (Stufe 2)
+Wetterwarnungen (Stufe 1)
+"""
+import logging
+import json
+import time
+import datetime
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_HOST, CONF_SSL, CONF_VERIFY_SSL, CONF_MONITORED_CONDITIONS)
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+_ENDPOINT = '/DWD/warnungen/warnapp_landkreise/json/warnings.json?jsonp=loadWarnings'
```

Comment:
how ca we fix this one? I cannot change the corresponding url, splitting it into multiple lines seems odd to me.